tools/pytorch2onnx.py

```diff
@@ -17,32 +17,25 @@
     raise NotImplementedError('please update mmcv to version>=v1.0.4')
 
 
-def pytorch2onnx(model,
+def pytorch2onnx(config_path,
```

Comment:
I think we'd better implement this function using ```generate_inputs_and_wrap_model```

Reply:
Thanks for the suggestion! I use the 2 helper functions instead of the `generate_inputs_and_wrap_model` because here we may want to keep the original model to verify whether the output is identical. If we use `generate_inputs_and_wrap_model` it may introduce some repetitive code to accomplish that.